frontend/src/tabs/zoom.pug

```diff
@@ -12,6 +12,13 @@
       span {{ info.sinceDate }} to {{ info.untilDate }} &nbsp;
       a(v-on:click="openSummary") [Show ramp chart for this period]
     .zoom__title--granularity.mini-font granularity: one ramp per {{ filterTimeFrame }}
+    .zoom__title--tags
+      template(v-for="commit in info.user.commits")
```

Comment:
Why not just use div?

Reply:
Reason is because using `template` does not introduce extra nodes in the DOM, whereas using `div` will. Since the functionality of these two nested `template` is to loop through the commits, it is unnecessary to create divs to complicate the DOM.

Reply:
OK